selfdrive/locationd/laikad.py

```diff
@@ -149,7 +150,10 @@ def read_ephemeris(self, gnss_msg):
         return
       ephem = parse_qcom_ephem(gnss_msg.drSvPoly, self.gps_week)
     else:
-      ephem = convert_ublox_ephem(gnss_msg.ephemeris)
+      if gnss_msg.which() == 'ephemeris':
+        ephem = convert_ublox_gps_ephem(gnss_msg.ephemeris)
+      else:
+        ephem = convert_ublox_glonass_ephem(gnss_msg.glonassEphemeris)
```

Comment:
i'd do an else if here and raise something like unknown msg type in the else block